boards/arm/samd5e5/metro-m4/src/sam_bq27426.c

```diff
@@ -0,0 +1,78 @@
+/****************************************************************************
+ * boards/arm/samd5e5/metro-m4/src/sam_bq27426.c
+ *
+ *   Copyright 2020 Falker Automacao Agricola LTDA.
```

Comment:
We cannot use our Copyright with Apache Foundation (too bad... I also liked the BSD way)